tests/polytope_test.py

```diff
@@ -48,5 +67,84 @@ def comparison_test():
         np.array([1, 0, 0])))
 
 
+def region_rotation_test():
+    p = pc.Region([pc.Polytope(A, b)])
+    p1 = pc.Region([pc.Polytope(A, b)])
+    p2 = pc.Region([pc.Polytope(Ab2[:, 0:2], Ab2[:, 2])])
+    p3 = pc.Region([pc.Polytope(Ab3[:, 0:2], Ab3[:, 2])])
+    p4 = pc.Region([pc.Polytope(Ab4[:, 0:2], Ab4[:, 2])])
+
+    p = p.rotation(0, 1, np.pi/2)
+    print(p.bounding_box)
+    assert(p == p2)
```

Comment:
I, too, used to enclose the asserted expression within parentheses. The parentheses can be omitted, because in Python `assert` is a [statement](https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement), not a function (in contrast for example to `print`, which was a statement in Python 2 and became a function in Python 3).